xskillscore/core/comparative.py

```diff
@@ -165,3 +172,146 @@ def _categorical_metric(observations, forecasts, dim):
     sign_test.coords["alpha"] = alpha
     sign_test.coords["confidence"] = confidence
     return sign_test
+
+
+def ttest_ind(a, b, dim=None, skipna=False, keep_attrs=False, equal_var=True):
+    """Calculate the T-test for the means of two independent samples of scores.
+
+    This is a two-sided test for the null hypothesis that 2 independent samples
+    have identical average (expected) values. This test assumes that the
+    populations have identical variances by default.
+
+    Parameters
+    ----------
+    a : xarray.Dataset or xarray.DataArray
+        Labeled array(s) over which to apply the function.
+    b : xarray.Dataset or xarray.DataArray
+        Labeled array(s) over which to apply the function.
+    dim : str, list
+        The dimension(s) to apply the correlation along. Note that this dimension will
+        be reduced as a result. Defaults to None reducing all dimensions.
+    skipna : bool
+        If True, skip NaNs when computing function.
+    keep_attrs : bool
+        If True, the attributes (attrs) will be copied
+        from the first input to the new one.
+        If False (default), the new object will
+        be returned without attributes.
+    equal_var : bool
+        If True (default), perform a standard independent 2 sample test that
+        assumes equal population variances [1]. If False, perform Welch’s
+        t-test, which does not assume equal population variance [2].
+
+    Returns
+    -------
+    statistic (float or array): The calculated t-statistics.
+    pvalue (float or array): The two-tailed p-value.
+
+    See Also
+    --------
+    scipy.stats.ttest_ind
+
+    References
+    ----------
+    https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ttest_ind.html
```

Comment:
I have a feeling this can be dropped because (hopefully) the see also provides a link to this page in rtd